.github/workflows/performance-regression.yml

```diff
@@ -15,6 +15,7 @@ env:
   specs: '{command:read,format:ion_text} {command:write,format:ion_text} {command:read,format:ion_binary} {command:write,format:ion_binary}'
   test_data_id: 'generated-test-data'
   run_cli: 'python amazon/ionbenchmark/ion_benchmark_cli.py'
+  run_cli_new: 'python src-python/amazon/ionbenchmark/ion_benchmark_cli.py'
```

Comment:
Should we just replace `run_cli` instead of `run_cli_new`?

Reply:
The new `run_cli_new` is temporary, to allow this PR to run the perf regression workflow. Since it pulls down the current HEAD which has the pre-PR layout, I need to run the benchmark CLI from that revision with `run_cli`.

Once this PR is merged, then I can follow up with a PR to replace `run_cli` with `run_cli_new`, so that the perf regression workflow continues working using the `src-python` layout for everything.

Reply:
Makes sense.